win32ss/user/ntuser/nonclient.c

```diff
@@ -494,41 +472,66 @@ DefWndDoSizeMove(PWND pwnd, WORD wParam)
              UserDrawMovingFrame( hdc, &sizingRect, thickframe );
 	  }
 
-	  if (msg.message == WM_KEYDOWN) UserSetCursorPos(pt.x, pt.y, 0, 0, FALSE);
-	  else
-	  {
-	      RECT newRect = unmodRect;
+      if (msg.message == WM_KEYDOWN)
```

Comment:
When I looked at the indentation here for the first time, it looked odd to me. But that is actually becuase the indentation is absolutely bonkers and confusing on the post-PR-line number 466
` if ( iconic ) /* ok, no system popup tracking */`
I know you didn't introduce this mess, but since you touch so much in here, I wonder whether you are willing to fix that mess in your PR?

Reply:
Yeah that file uses 3, 4 and who knows how many spaces in various places and some random tabs just to spice it up. I tried to follow the existing code in the places I had to touch it for the most part. I have "show whitespace" on in my editor and I changed some silly tabs to spaces while still trying to indent by 3 when the rest of the code did and I did not change everything inside "that" brace.

I don't want to touch more code than I need to in this PR.

Reply:
I checked: It would just require to touch 21 additional lines in this function to eliminate all the tabs in here and to have pure space indentation everywhere. Which woul make future reviews much easier.

Reply:
I can eliminate the tabs in this function but not 3->4 spaces which is not appropriate for this PR IMHO.

Reply:
I do agree to your proposal. This is exactly what I had in mind, and what makes most sense.